distribution/lib/Standard/Base/0.0.0-dev/src/Logging.enso

```diff
@@ -12,13 +12,13 @@ polyglot java import java.util.logging.Logger
 Any.log_message : Text -> Log_Level -> Any
 Any.log_message self ~message:Text level:Log_Level=Log_Level.Info =
     type_name = Meta.get_qualified_type_name self
-    logger = Logger.getLogger type_name
+    logger = LoggerFactory.getLogger type_name
     case level of
-        Log_Level.Finest -> logger.finest message
-        Log_Level.Fine -> logger.fine message
+        Log_Level.Finest -> logger.trace message
+        Log_Level.Fine -> logger.trace message
```

Comment:
Shouldn't `Finest` and `Fine` be distinct levels? `trace` and `debug` respectively.

```suggestion
        Log_Level.Fine -> logger.debug message
```

Reply:
indeed